server/rooms.ts

```diff
@@ -1279,6 +1279,28 @@ export class GlobalRoomState {
 		let count = 0;
 		if (!Config.usepostgres) return 0;
 		const logDatabase = new PostgresDatabase();
+		let [{value}] = await logDatabase.query(`SELECT value FROM battle_db_info WHERE key = 'version'`).catch(
+			() => [{value: "0"}]
+		);
+		value = Number(value);
+		const files = FS(`databases/migrations/storedbattles`)
+			.readdirSync()
+			.map(f => Number(f.slice(1).split('.')[0]));
```

Comment:
```suggestion
			.filter(f => f.endsWith('.sql'))
			.map(f => Number(f.slice(1).split('.')[0]));
```
Stray files like `.DS_Store` or an editor swap file could cause problems.